multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/PollIncrementalAppInstanceUpdateExecution.java

```diff
@@ -54,15 +54,15 @@ private void updateAsyncExecutionIndexes(ProcessContext context) {
                              .equals(CloudApplication.State.STOPPED)) {
             continueWithIncrementalScaling(context);
         } else {
-            pollApplicationIndexesBeforeContinueingWithIncrementalScaling(context);
+            pollApplicationIndexesBeforeContinuingWithIncrementalScaling(context);
         }
     }
 
     private void continueWithIncrementalScaling(ProcessContext context) {
         context.setVariable(Variables.ASYNC_STEP_EXECUTION_INDEX, 2);
     }
 
-    private void pollApplicationIndexesBeforeContinueingWithIncrementalScaling(ProcessContext context) {
+    private void pollApplicationIndexesBeforeContinuingWithIncrementalScaling(ProcessContext context) {
```

Comment:
This method name seems too long. Can it be simplified?

Reply:
Renamed it